src/test/java/com/aws/greengrass/lifecyclemanager/GreengrassServiceTest.java

```diff
@@ -74,6 +90,7 @@ void beforeEach() throws IOException, URISyntaxException, ServiceLoadException {
     @AfterEach
     void afterEach() throws IOException {
         context.close();
+        executorService.shutdown();
```

Comment:
shutdownnow

Reply:
Got it, I used shutdownNow before but modified to shutdown
Is there a reason to prefer shutdownNow over shutdown?

Reply:
shutdown now kills anything that's currently running, we don't want anything stray to keep running.